db/migrate/20200617014320_add_subscriber_email_to_user_subscriptions.rb

```diff
@@ -0,0 +1,5 @@
+class AddSubscriberEmailToUserSubscriptions < ActiveRecord::Migration[6.0]
+  def change
+    add_column :user_subscriptions, :subscriber_email, :string, null: false
```

Comment:
The `null: false` without a `default` should be fine here since there are no records in Production.